editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -4359,6 +4380,27 @@ void SpatialEditor::_menu_item_toggled(bool pressed, int p_option) {
 			tool_option_button[TOOL_OPT_USE_SNAP]->set_pressed(pressed);
 			snap_enabled = pressed;
 		} break;
+
```

Comment:
Ah I understand what you want to do now, but using an "active" property is terrible,
just make it more explicit, like "replicating_camera_in_debugger" or something like that.

Reply:
So you mean just rename the variable? I agree `active` isn't self-explanatory at all...

Reply:
Oh, now I recall why I named it `active`. This is a state of viewport not necessarily tied to the debugger. It just flags the most recently used 3D viewport. It works even if the replication is not turned on. Admittedly, the name is still not good and should be renamed. Maybe name it `last_used_viewport` and move it to the `SpatialEditor` as an int index of last used viewport since it's always just one viewport. Either way, I thought this could be used for other functionality, not only the debugger.

Reply:
I would still make it explicit since its only used for this. No need to future proof it.